src/utils/bv-event.class.js

```diff
@@ -26,7 +26,7 @@ export default class BvEvent {
     // Create a private variable using closure scoping.
     let defaultPrevented = false
     // Recreate preventDefault method. One way setter.
-    this.preventDefault = function preventDefault() {
+    this.preventDefault = function() {
```

Comment:
The one thing about giving functions names, is that the name will appear in stack traces, rather than appearing as `anonymous function`.  Although that is mainly handy for larger inline functions.